scorecard/profile_data/indicators/capital_budget_spending.py

```diff
@@ -0,0 +1,99 @@
+from .indicator_calculator import IndicatorCalculator
+from .utils import (
+    group_by_year,
+    populate_periods,
+    filter_for_all_keys,
+    percent,
+)
+
+
+class CapitalBudgetSpending(IndicatorCalculator):
```

Comment:
meer technically is dit "capital spending budget vs actual' of "capital over/under-spending"

Reply:
spending beteken nie noodwendig actual nie - although daar is een of ander convention van expenditure vs spending 

So eks nie seker hierdie naam is duideliker nie.

Reply:
wat van CapitalOverUnderSpending

ons wil spending in die naam he want dis nie revenue/income nie
ons wil die notion van over/under of budget vs actual (nie heeltemal nie) of difference daar he.
ek dink over/under is die mees duidelike opsie.

Reply:
@jbothma my thinking here (which is a rule I try to apply consistently, within reason) is to keep the names that we use on the back-end as close as possible to what we see on the front-end. Would you still like me to adjust then names to resemble more closely to what they actually represent?

Reply:
yeah - I think the top priority is that the code is clear about what it represents - this change is making it less clear.

I agree with backend and frontend names matching, so I would propose we call this `CapitalOverUnderSpend` now - intentionally capitalising S so it's clearer that it relates to both the over and under, and if you would like to make frontend and backend names match, rather get it fixed in the frontend when you and/or matt have time, than make the backend match the unclear frontend name.
ping @juriejan

Reply:
same for operating over/under-spend section

Reply:
Makes sense @jbothma. I'll create a task for reviewing the titles of the indicators.